lib/operators/indent-operators.coffee

```diff
@@ -6,7 +7,10 @@ class AdjustIndentation extends Operator
     @motion.select(count)
     {start} = @editor.getSelectedBufferRange()
 
-    @indent()
+    if mode is 'visual'
+      _.times(count, => @indent())
```

Comment:
needs `@editor.transact` before `_.times` - you may see that in `general-operators.coffee` - otherwise you need to press `u` twice to undo `v2>` .